Neos.Flow/Classes/Reflection/ReflectionService.php

```diff
@@ -996,6 +997,18 @@ public function getPropertyTagValues($className, $propertyName, $tag)
         return (isset($this->classReflectionData[$className][self::DATA_CLASS_PROPERTIES][$propertyName][self::DATA_PROPERTY_TAGS_VALUES][$tag])) ? $this->classReflectionData[$className][self::DATA_CLASS_PROPERTIES][$propertyName][self::DATA_PROPERTY_TAGS_VALUES][$tag] : [];
     }
 
+    /**
+     * Returns the property type
+     *
+     * @param $className
+     * @param $propertyName
+     * @return string|null
+     */
+    public function getPropertyType($className, $propertyName)
```

Comment:
```suggestion
    public function getPropertyType(string $className, string $propertyName): ?string
```
Anything speaking against typehinting this new method?

Reply:
Well, only that all the other methods in the Reflection Service also should have proper type hints … so I just did what the surrounding code did, so that it fits. To me it doesn't matter if we add the type hints now or when we clean up the whole class properly in a dedicated change.